cristin-project/src/test/java/no/unit/nva/cristin/projects/project.feature

```diff
@@ -99,80 +99,42 @@ Feature: API tests for Cristin Project retrieve and search
     * def uri = 'https://api.dev.nva.aws.unit.no/' + basePath + '/project/0'
     And match response.detail == "The requested resource '" + uri + "' was not found"
 
-  Scenario: Fetch returns all supported fields
-    Given path '/project/6720765'
+  Scenario: Fetch returns supported fields
```

Comment:
Reducing karate code as we can test a lot of fields using only one project from identifier